Packs/AzureDataExplorer/ReleaseNotes/1_2_9.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Azure Data Explorer
+- Added the ***azure-data-explorer-generate-login-url*** command to easy achieve the auth code.
```

Comment:
@ShirleyDenkberg was this reviewed? This doesn't seem right.
I feel it should be more like this, please confirm.
```suggestion
- Added the ***azure-data-explorer-generate-login-url*** command to easily get the auth code.
```